src/geometry/manifold/instances/units_of_normed_algebra.lean

```diff
@@ -0,0 +1,54 @@
+/-
+Copyright © 2021 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri, Heather Macbeth
+-/
+
+import geometry.manifold.algebra.lie_group
+import analysis.normed_space.units
+
+/-!
+# Units of normed algebra
+
+In this file we aim to prove that the units of a complete normed algebra are a Lie group.
+
+## TODO
+
+We are still missing the main result, i.e. the instance
+
+```
+instance units_of_normed_algebra.lie_group
+  {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+  {R : Type*} [normed_ring R] [normed_algebra 𝕜 R] [complete_space R] :
+  lie_group (model_with_corners_self 𝕜 R) (units R) :=
+{ smooth_mul := begin
+    sorry,
+  end,
+  smooth_inv := begin
+    sorry,
+  end,
+  ..units.smooth_manifold_with_corners,
+  /- Why does it not find the instance alone? -/ }
+```
+-/
+
+noncomputable theory
+
+open_locale manifold
+
+instance {R : Type*} [normed_ring R] [complete_space R] : charted_space R (units R) :=
+units.open_embedding_coe.singleton_charted_space
```

Comment:
If you put this instance and the other instance inside the `units` namespace, you won't need to reintroduce variables every time -- just have 
```lean
variables {R : Type*} [normed_ring R] [complete_space R]
```
at the start and 
```
{𝕜 : Type*} [nondiscrete_normed_field 𝕜] [normed_algebra 𝕜 R]
```
for the last instance.

Reply:
I fixed this.